app/controllers/spina/new_register_controller.rb

```diff
@@ -0,0 +1,62 @@
+require 'deskpro_feedback'
+require 'json'
+
+module Spina
+  class NewRegisterController < Spina::ApplicationController
+
+    layout 'layouts/default/application'
+
+    def index
+    end
+
+    def select_request_or_suggest
+      if params[:subject] == 'request'
+        redirect_to new_register_request_register_path
+      else
+        redirect_to new_register_suggest_register_path
+      end
+    end
+
+    def suggest_register
+      @new_register = Spina::NewRegister.new
+    end
+
+    def request_register
+      @new_register = Spina::NewRegister.new
+    end
+
+    def thanks
+    end
+
+    def create
+      @new_register = Spina::NewRegister.new(new_register_params)
+
+      if @new_register.valid?
+        @deskproService = DeskproFeedback.new(new_register_params)
+        @response = @deskproService.send_feedback
+        if @response.status.code != 201
+          logger.debug("Suggest register ticket creation failed with status: #{@response.status} and content: #{@response.body}")
```

Comment:
I'm pretty sure the standard log level in production is info so this log wont go into the logs most likely. Generally you would use the error level for this.